src/ui/Controllers/CoursesController.cs

```diff
@@ -19,16 +19,20 @@ public CoursesController(ManageApi manageApi)
             _manageApi = manageApi;
         }
 
-        public async Task<IActionResult> Index()
+        [Route("{organisationId}")]        
+        public async Task<IActionResult> Index(string organisationId)
         {
-            var courses = await _manageApi.GetCourses();
-            var data = await _manageApi.GetOrganisationCoursesTotal();
+            var courses = await _manageApi.GetCoursesByOrganisation(organisationId);
+            var orgs = await _manageApi.GetOrganisations();
+
+            var data = await _manageApi.GetOrganisationCoursesTotal(organisationId);
             var model = new CourseListViewModel
             {
                 Courses = courses,
-                TotalCount = data.TotalCount
+                TotalCount = data.TotalCount,
+                MultipleOrganisations = orgs.Count() > 1
             };
-            return View(model);
+            return View("Index", model);
```

Comment:
Is there an actual need for string "index"? 
`return View(model);` that should suffice